### ISSUE TITLE
Update to phystrix 2.x

### ISSUE THREAD

**Issue by Henridv** [odesk/phystrix-bundle#9]
Can this bundle be updated to use odesk/phystrix 2.x?

**Comment by lcf** [comment]
yes, it is probably just a matter of updating one digit in composer.json
feel free to try and submit a PR if you like @Henridv
